src/macCamera.py

```diff
@@ -1,46 +1,69 @@
 import cv2
 import numpy as np
 import time
+from zones import Zone
 
 def nothing(x):
     pass
 
-cap = cv2.VideoCapture(0)
-
-fgbg = cv2.createBackgroundSubtractorMOG2()
+saveImage = False
+imageNumber = 0
+zone1 = Zone()
 
+cap = cv2.VideoCapture(0)
 _, frame = cap.read()
 w, h, _ = frame.shape
 
+fgbg = cv2.createBackgroundSubtractorMOG2()
+
 time.sleep(0.1)
 
+# window setup
+cv2.namedWindow('frame')
 cv2.namedWindow('heatMap')
+
 cv2.createTrackbar('Subtract', 'heatMap', 1, 100, nothing)
 
-# print subtract
-heatMap = np.zeros((w/2, h/2), dtype=np.uint64)
+# setup callbacks
+cv2.setMouseCallback('frame', zone1.setSquare)
+
+heatMap = np.zeros((w/2, h/2), dtype=np.uint8)
 
 while True:
     _, frame = cap.read()
     w, h, _ = frame.shape
     frame = cv2.resize(frame, (h/2,w/2))
 
     subValue = float(cv2.getTrackbarPos('Subtract', 'heatMap') / 100)
-    # frameNoiseReduction = cv2.fastNlMeansDenoisingColored(frame, None , 10, 10, 7, 21)
+
     subtract = np.full((w/2, h/2), subValue, dtype=np.uint8)
```

Comment:
is this subtracting the outside edges of the frame?

Reply:
this is the subtraction layer. At this time it is not in use and will need to be cleaned up.